gusto/solvers/linear_solvers.py

```diff
@@ -383,6 +387,9 @@ def update_reference_profiles(self):
             logger.info('Compressible linear solver: Exner average solve')
             self.exner_avg_solver.solve()
 
+        self.hybridized_solver.invalidate_jacobian()
```

Comment:
We discussed offline adding a comment here to remind us of what this does

Reply:
added